net-libs/libmodbus/libmodbus-3.0.8.recipe

```diff
@@ -0,0 +1,84 @@
+SUMMARY="A groovy modbus library"
+DESCRIPTION="
```

Comment:
Start the description on the same line (not on a new line) (guidelines)

Reply:
ok updated, thanks @Begasus